bc_obps/registration/tests/endpoints/test_useroperator_endpoints.py

```diff
@@ -1095,6 +1095,62 @@ def test_put_user_operator_operator(self):
         assert operator is not None
         assert operator.updated_by == self.user
         assert operator.updated_at is not None
+        assert mock_payload == {
+            "legal_name": operator.legal_name,
+            "trade_name": operator.trade_name,
+            "cra_business_number": operator.cra_business_number,
+            "bc_corporate_registry_number": operator.bc_corporate_registry_number,
+            "business_structure": operator.business_structure.pk,
+            "physical_street_address": operator.physical_address.street_address,
+            "physical_municipality": operator.physical_address.municipality,
+            "physical_province": operator.physical_address.province,
+            "physical_postal_code": operator.physical_address.postal_code,
+            "mailing_address_same_as_physical": operator.mailing_address_id == operator.physical_address_id,
+            "mailing_street_address": operator.mailing_address.street_address,
+            "mailing_municipality": operator.mailing_address.municipality,
+            "mailing_province": operator.mailing_address.province,
+            "mailing_postal_code": operator.mailing_address.postal_code,
+            "operator_has_parent_operators": operator.parent_operators.exists(),
+            "parent_operators_array": list(operator.parent_operators.all()),
+        }
+
+    def test_put_user_operator_operator_with_an_existing_cra_business_number(self):
+        existing_operator = operator_baker()
+        new_operator = operator_baker()
+        new_operator.created_by = self.user
+        new_operator.save(update_fields=["created_by"])
```

Comment:
The `operator_baker` function now takes a `custom_properties` arg, so instead of lines 1120-1121, you could use that, e.g. something like new_operator = operator_baker({created_by: self.user})